packages/client/src/services/organizations.js

```diff
@@ -0,0 +1,54 @@
+
+export async function getOrganizations() {
+
+  const url = `${process.env.REACT_APP_FIRESTORE_API_BASE_URL}organizations`;
```

Comment:
Since this doesn't have anything that we really need to hide - should we create a .env file?

Reply:
From what I read online putting on .env can only make it accessible on build, failed to run this when running on npm run start.
But I'm always careful of exposing anything on a public repo :/

Reply:
ok that's fine we can add it as a setting